test/unit/bind-radio-request.spec.js

```diff
@@ -76,14 +76,14 @@ describe('Marionette.bindRadioRequests', function() {
   describe('when bindings is not an object', function() {
     beforeEach(function() {
       this.run = function() {
-        Marionette.bindRadioRequests(this.target, this.channel, 'replyFooStub');
+        Marionette.bindRequests(this.target, this.channel, 'replyFooStub');
       }.bind(this);
     });
 
     it('should error', function() {
       expect(this.run).to.throw(Marionette.Error, new Marionette.Error({
         message: 'Bindings must be an object.',
-        url: 'marionette.functions.html#marionettebindradiorequests'
+        url: 'marionette.functions.html#marionettebindRequests'
```

Comment:
again these'll match links on the website that are generated from the `docs/*.md` and so I think those `#hash` are all lower case.

That does point out something though.. the docs need to be updated as well to reflect these changes.